packages/fast-jss-manager-react/src/jss-manager.tsx

```diff
@@ -14,8 +14,16 @@ import { designSystemContext } from "./context";
  * Describes an interface for adjusting a styled component
  * per component instance
  */
-export interface JSSManagedComponentProps<S, C> {
+export interface JSSManagedComponentProps<T, S, C> {
+    /**
+     * The component instance stylesheet
+     */
     jssStyleSheet?: Partial<ComponentStyles<S, C>>;
+
+    /**
+     * React reference to the component instance managed by the JSSManager
+     */
+    innerRef?: React.LegacyRef<React.Component<T & ManagedClasses<S>>>;
```

Comment:
Good call using the broadest definition. :clap:

In other words, so interesting that the union type of `LegacyRef` includes both the legacy string ref and the current implementation. IMO, that is the least intuitive name to include those under...